cmd/storagenode-updater/main.go

```diff
@@ -203,19 +205,15 @@ func update(ctx context.Context, binPath, serviceName string, renameBinary renam
 	}
 	log.Println("finished downloading", downloadURL, "to", tempArchive.Name())
 
-	err = renameBinary(currentVersion)
-	if err != nil {
-		return errs.Wrap(err)
-	}
-
-	err = unpackBinary(ctx, tempArchive.Name(), binPath)
+	newVersionPath := prependExtension(binPath, currentVersion.String())
```

Comment:
Shouldn't we prepend the `suggestedVersion` instead of the `currentVersion`?

Reply:
Oops.. yea that would make sense. The weird thing is that when I tested this it did what I expected.

Reply:
I guess it doesn't change the result as that path is only transient. Will fix though.